exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,10 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+let preciseStudentPercentage = (numberOfStudents / 23) * 100;
```

Comment:
Works as expected , well done.
As a suggestion, try to avoid [magic numbers](https://www.pluralsight.com/tech-blog/avoiding-magic-numbers/) in your code. 
Would the _preciseStudentPercentage_ still be correct if the _numberOfStudents_ or _numberOfMentors_ changes ?
In this scenario, 23 is a "magic number". We know that it works now because that is the total of students + mentors, but what can we do to make it work if those values change?